avocado/utils/network/interfaces.py

```diff
@@ -571,3 +571,29 @@ def restore_slave_cfg_file(self):
                     os.remove(slave_config)
             except Exception as ex:
                 raise NWException(f"Could not restore the config file {ex}")
+
+    def are_packets_lost(self, peer_ip, count, option=None, flood=False):
```

Comment:
Here you are using `count` and `flood` as arguments, but using `option` for other arguments. What was the criteria used here? I understand having all the ping options as arguments here is not practical, however mixing this styles is also confusing from the user PoV. So my suggestion is to move `count` and `flood` to `options`, since they are actually options for the ping command.

Reply:
Kept Only 2 aurguments , count and options as explained in above comment.